src/embind/emval.js

```diff
@@ -132,6 +132,13 @@ var LibraryEmVal = {
     return __emval_register(v);
   },
 
+  _emval_new_spread__deps: ['_emval_register', '$requireHandle'],
+  _emval_new_spread: function(constructor, args) {
+    constructor = requireHandle(constructor);
+    args = requireHandle(args);
+    return __emval_register(new constructor(...args));
```

Comment:
This would be another situation where embind would require running in a browser that supports ES6?

@jlee-imvu Is that a safe assumption at IMVU?

@kripken What do you think?

Reply:
We shouldn't add ES6 notation, for now - eventually we'll update our optimizer to handle it, and after that we'd need to have support for compiling down to ES5. Until then, we do still want to support older browsers and VMs.